cluster/configs/src/main/java/com/linkedin/openhouse/cluster/configs/TblPropsToggleRegistry.java

```diff
@@ -0,0 +1,31 @@
+package com.linkedin.openhouse.cluster.configs;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import javax.annotation.PostConstruct;
+import org.springframework.stereotype.Component;
+
+/** A central registry for all table properties that are managed through Feature Toggle. */
+@Component
```

Comment:
why must this class be spring managed? 

as it is defined here, the registry can be reduced to a constant map. is there a future vision to extend the registry?

Reply:
That's basically all benefits that dependency injection can bring: ease of testing, no constructor change etc. 
And yes We might want to extend this behavior in the internal app.